src/instructlab/model/test.py

```diff
@@ -30,54 +38,128 @@
+                f"training only. - {adapter_file}"
+            )
 
-    print("system prompt:", utils.get_sysprompt())
-    for idx, example in enumerate(test_data):
-        system = example["system"]
-        user = example["user"]
-        print("[{}]\n user prompt: {}".format(idx + 1, user))
-        prompt = f"<|system|>\n{system}\n<|user|>\n{user}\n<|assistant|>"
-        print("expected output:", example["assistant"])
+        # Load the JSON Lines file
+        test_data_dir = f"{data_dir}/test.jsonl"
+        if not os.path.exists(test_data_dir):
+            click.secho(
+                f"'{test_data_dir}' not such file or directory. Did you run 'ilab model train'?",
+                fg="red",
+            )
+            raise click.exceptions.Exit(1)
+        with open(test_data_dir, "r", encoding="utf-8") as f:
+            test_data = [json.loads(line) for line in f]
 
-        print("\n-----model output BEFORE training----:\n")
-        load_and_train(model=model_dir, no_adapter=True, max_tokens=100, prompt=prompt)
+        print("system prompt:", utils.get_sysprompt())
+        for idx, example in enumerate(test_data):
+            system = example["system"]
+            user = example["user"]
+            print("[{}]\n user prompt: {}".format(idx + 1, user))
+            prompt = f"<|system|>\n{system}\n<|user|>\n{user}\n<|assistant|>"
+            print("expected output:", example["assistant"])
 
-        if adapter_file_exists:
-            print("\n-----model output AFTER training----:\n")
+            print("\n-----model output BEFORE training----:\n")
             load_and_train(
-                model=model_dir,
-                adapter_file=adapter_file,
-                max_tokens=100,
-                prompt=prompt,
+                model=model_dir, no_adapter=True, max_tokens=100, prompt=prompt
+            )
+
+            if adapter_file_exists:
+                print("\n-----model output AFTER training----:\n")
+                load_and_train(
+                    model=model_dir,
+                    adapter_file=adapter_file,
+                    max_tokens=100,
+                    prompt=prompt,
+                )
+    else:
+        logger.debug("")
+        # pylint: disable=import-outside-toplevel
+        # Local
+        from .linux_test import linux_test
+
+        if not test_file:
+            try:
+                test_file = sorted(
+                    Path(ctx.obj.config.generate.output_dir).glob("test_*"),
+                    key=os.path.getmtime,
+                    reverse=True,
+                )[0]
+            except Exception as exc:
+                click.secho("No test files found", fg="red")
+                raise click.exceptions.Exit(1) from exc
+        try:
+            answers = linux_test(
+                ctx,
+                test_file,
+                models=[model, Path("models") / "ggml-model-f16.gguf"],
```

Comment:
Where is this `ggml-model-f16.gguf` coming from?

Reply:
It is coming from 
`src/instructlab/model/train.py:353:        gguf_models_file = gguf_models_dir / "ggml-model-f16.gguf"`
ideally it should be parameterized as in
`tests/test_lab_train.py:27:LINUX_GGUF_FILE = FINAL_RESULTS_DIR + "/ggml-model-f16.gguf"`